command/git.go

```diff
@@ -168,6 +173,9 @@ func (opts cloneOpts) buildHTTPCloneCommand() ([]string, error) {
 
 func (opts cloneOpts) buildSSHCloneCommand() ([]string, error) {
 	cloneCmd := fmt.Sprintf("git clone '%s' '%s'", opts.location, opts.dir)
+	if !opts.mergeTestRequester && opts.shallowClone {
+		cloneCmd = fmt.Sprintf("%s --depth 1", cloneCmd)
```

Comment:
would a greater depth reduce the frequency of needing to `--unshallow`?

Reply:
Oh good question. Thanks for reminding me. I did experiments of cloning mongodb/mongo and 10gen/mms on a spawn host. Ordinary clones took about 40 seconds. Shallow clones took about 14 seconds for depth 1, 10, and 100, and started getting somewhat slower for 1000. I've switched this to 100.